src/main/java/edu/umich/its/cpm/MigrationTaskService.java

```diff
@@ -1088,9 +1088,11 @@ private StringBuffer preMigrationChecks(StringBuffer itemStatus,
 				BoxFolder folder = new BoxFolder(api, boxFolderId);
 				log.info("upload file " + fileName + " size " + fileSize + " to folder " + folder.getID());
 
+				fileName = Utils.sanitizeName(type, fileName);
```

Comment:
This should just be added to sanitizeName.  Otherwise it gets much harder for someone to understand how names need to get adjusted.  That makes it much harder to reuse the code.

Do we really need both sanitizeName and sanitizeFolderNames?